go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/submariner-io/lighthouse v0.15.0-m2.0.20230130153953-7107bd35c649
 	github.com/submariner-io/shipyard v0.15.0-m2
 	github.com/submariner-io/submariner v0.15.0-m2
-	github.com/submariner-io/submariner-operator v0.15.0-m2
+	github.com/submariner-io/submariner-operator v0.15.0-m2.0.20230206134304-33b31d9d2503
```

Comment:
This will need to be updated once the operator patch is merged.

Reply:
Yes - I was planning to do that.

Reply:
I thought you did — I just wanted to make sure the PR didn’t get two approvals and merged before it was actually ready.

Reply:
It's updated...